enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/UnderUtdanning.kt

```diff
@@ -7,4 +7,11 @@ data class UnderUtdanning(val skoleUtdanningssted: Søknadsfelt<String>,
                           val hvorMyeSkalDuStudere: Søknadsfelt<Int>?,
                           val hvaErMåletMedUtdanningen: Søknadsfelt<String>?,
                           val utdanningEtterGrunnskolen: Søknadsfelt<Boolean>,
-                          val tidligereUtdanninger: Søknadsfelt<List<Utdanning>>? = null)
+                          val tidligereUtdanninger: Søknadsfelt<List<Utdanning>>? = null,
+                          val semesteravgift: Søknadsfelt<Double>? = null,
+                          val studieavgift: Søknadsfelt<Double>? = null,
+                          val eksamensgebyr: Søknadsfelt<Double>? = null)
```

Comment:
Jeg la inn de feltene her. Notere at utdanning ligger på rootnivå på SøknadSkolepenger. For Overgangsstønad ligger det under aktivitet. 
Hva tenker dere?